pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -100,7 +101,17 @@ BEGIN
         END IF;
 
         IF SCHEMA_CATALOG.is_timescaledb_installed() AND SCHEMA_CATALOG.get_default_compression_setting() THEN
-            PERFORM SCHEMA_PROM.set_compression_on_metric_table(r.table_name, TRUE);
+            IF SCHEMA_CATALOG.get_timescale_major_version() >= 2 THEN
```

Comment:
We should create an is_distributed function because this code repeats here and at line 157